concent_api/core/views.py

```diff
@@ -378,6 +387,16 @@ def handle_send_force_report_computed_task(request, client_message):
         requestor_public_key = other_party_public_key,
         status               = ReceiveStatus
     )
+    store_subtask(
+        task_id              = client_message.report_computed_task.task_to_compute.compute_task_def['task_id'],
+        subtask_id           = client_message.report_computed_task.task_to_compute.compute_task_def['subtask_id'],
+        provider_public_key  = client_public_key,
+        requestor_public_key = other_party_public_key,
+        state                = Subtask.SubtaskState.FORCING_REPORT,
+        next_deadline        = client_message.report_computed_task.task_to_compute.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME,
```

Comment:
@cameel Please check if `next_deadline` is defined correctly and if correct related messages are stored.